docs/installation/standalone.mdx

```diff
@@ -0,0 +1,417 @@
+---
+title: Standalone Installation
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+## Overview
+
+This documents will show you how to install KubeVela in standalone mode. There are two options:
+
+1. [Local machine](#local): Install in a local machine.
+2. [Remote Linux server](#remote): Install on a remote Linux server with accessible IP.
+
+### About VelaD
+
+We'll use [VelaD](https://github.com/kubevela/velad) to install KubeVela in standalone mode. VelaD is a CLI tool that
+packages KubeVela with all dependencies for minimal installation and VelaUX.
+VelaD enables you to run KubeVela on any of docker compatible environments or Linux system.
+
+- VelaD provides Kubernetes by leveraging [K3s](https://rancher.com/docs/k3s/latest/en/quick-start/) on linux or [k3d](https://k3d.io/) on docker environment.
+- KubeVela along with all related images, and `vela` command line are packaged together that enables air-gapped installation of KubeVela and VelaUX.
+
+
+:::caution
+Installation with standalone mode is suited for development and testing purpose only. It is not recommended to use it in production environment.
+:::
+
+## Install in a local machine {#local}
+
+### 1. Prepare environment
+
+You can install KubeVela on machines with these OS: Linux, macOS, Windows。
+
+**Requirements**
+
+1. Make sure your machine have `curl` installed.
```

Comment:
does windows need it?

Reply:
yes, AFAIK, powershell have "curl" inside.

Reply:
> yes, AFAIK, powershell have "curl" inside.

I'm wrong. Powershell have another command for http request.